src/components/LinterMessage/styles.module.scss

```diff
@@ -1,3 +1,43 @@
+$top-arrow-width: 10px;
+$top-arrow-height: 20px;
+
+.inline:first-child {
+  &:after,
+  &:before {
+    border-color: transparent;
+    border-style: solid;
+    content: '';
+    display: block;
+    height: 0;
+    left: 10px;
+    position: absolute;
+    width: 0;
+  }
+
+  &:after {
+    border-width: $top-arrow-width;
+    top: -$top-arrow-height + 2;
+  }
+
+  &:before {
+    border-bottom-color: inherit;
+    border-width: $top-arrow-width;
+    top: -$top-arrow-height;
+  }
+}
+
+.danger.inline:after {
+  border-bottom-color: #f8d7da;
```

Comment:
Is there a way to import the colors directly? Otherwise, I think it would help to move these to color variables just so that we can see them all in one place.

Reply:
It is a mess. It uses two or three different SCSS mixins, and the final color is derived from another set of colors that is different :/

Reply:
oh! I was more successful this evening, and I found how to use bootstrap code (by reading the sources, again)